distribution/src/dist/etc/ufw/applications.d/openfire

```diff
@@ -0,0 +1,39 @@
+[Openfire C2S]
+title=Openfire XMPP Server - Client to Server (C2S) connections
+description=Allow for XMPP clients to connect to the server and also use File Transfers
+ports=5222|5223|8010|7777
```

Comment:
Given that the ports are configurable, wouldn't these need to be set at runtime?